node/make.js

```diff
@@ -48,6 +46,7 @@ target.test = function() {
     cp('-Rf', rp('test/scripts'), testPath);
     cp('-Rf', rp('test/fakeTasks'), testPath);
     process.env['TASKLIB_INPROC_UNITS'] = '1'; // export task-lib internals for internal unit testing
+    set('+e');
```

Comment:
What's this for? Can we add a comment?

Reply:
Basically it restores the old behavior of shelljs before the upgrade. It avoids throwing an exception when tests fail and instead just reports that it failed.